include/gridtools/storage/builder.hpp

```diff
@@ -38,10 +38,15 @@ namespace gridtools {
             void initializer_impl(
                 Fun const &fun, T *dst, Layout layout, info<N> const &info, std::index_sequence<Is...>) {
                 int length = info.length();
+                auto &&lengths = info.lengths();
 #pragma omp parallel for
                 for (int i = 0; i < length; ++i) {
                     auto indices = info.indices(layout, i);
+                    for (auto ok : {(indices[Is] < lengths[Is])...})
```

Comment:
I thought I commented already yesterday, but apparently didn't commit.
I appreciate that I learned a new pattern (range-for-loop over initializer list created from pack expansion)!
However, I'd prefer a more expressive way for sake of maintainability by outlining to a named function/lambda, such that it reads something like
```
if(in_range(...))
   dst[i] = fun(indices[Is]...);
```

Reply:
done